.travis.yml

```diff
@@ -36,16 +37,17 @@ matrix:
     - env: HIVE_TESTS=true
     - env: KUDU_TESTS=true
 
+    # HDP 3
+    - if: type IN (push, cron)
```

Comment:
Do we have `cron` configured?

Does `push` mean pull request?

Reply:
`push` mean

Reply:
> Do we have `cron` configured?

Doesn't matter, does it?

> Does `push` mean pull request?

it means "push to a branch"

`pull_request` means "pull request". 

maybe I should use negation here? `type != pull_request` ?